chain-impl-mockchain/Cargo.toml

```diff
@@ -9,21 +9,21 @@ authors = [ "Nicolas Di Prima <nicolas.diprima@iohk.io>"
           ]
 edition = "2018"
 
+[features]
+generic-serialization = [ "cardano/generic-serialization", "chain-addr/generic-serialization", "serde", "serde_derive" ]
+
 [dependencies]
-serde = "^1.0"
-serde_derive = "^1.0"
-bincode = "^1.0.1"
-quickcheck = { version = "0.8" }
-chain-core = { path = "../chain-core" }
-chain-addr = { path = "../chain-addr" }
-rand = "0.6"
 num-traits = "0.2"
 num-derive = "0.2"
+serde = { version = "^1.0", optional = true }
+serde_derive = { version = "^1.0", optional = true }
+cardano = { path = "../cardano" }
+chain-core = { path = "../chain-core" }
+chain-addr = { path = "../chain-addr" }
 
-[dependencies.cardano]
-path = "../cardano"
-features = [ "generic-serialization" ]
 
 [dev-dependencies]
+quickcheck = "0.8"
+rand = "0.6"
 chain-core = { path = "../chain-core", features=["property-test-api"]}
 chain-addr = { path = "../chain-addr", features=["property-test-api"]}
```

Comment:
I wonder if it's bad to have these duplicated as dev-dependencies with a different feature set after this change.

Reply:
At the state of this commit they are needed only as dev-dependencies. Are they going to be used as normal dependencies too?

Having the same dependency in both places with different features works well, I've done this in other project.

Reply:
> Are they going to be used as normal dependencies too?

I don't know, it's this PR that tries to list them also as normal dependencies for some reason?

Reply:
No, this PR moves them from normal deps to dev-deps

Reply:
Eh, I'm talking about chain-core and chain-addr.